include/lsst/modelfit/parameters/object.h

```diff
@@ -21,12 +22,12 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#ifndef PARAMETERS_OBJECT_H
-#define PARAMETERS_OBJECT_H
+#ifndef LSST_MODELFIT_PARAMETERS_OBJECT_H
+#define LSST_MODELFIT_PARAMETERS_OBJECT_H
 
 #include <string>
 
-namespace parameters {
+namespace lsst::modelfit::parameters {
 
 class Object {
```

Comment:
I'll add docs. I named this class `Object` because I figured I might have a use for a generic base class for any object (parameter, transform, whatever), even if just for Python type hints. I never ended up needing anything beyond str/repr or storing Objects in a single C++ container.